src/lib/consts/displayTexts.ts

```diff
@@ -40,11 +41,13 @@ export enum EFilterPanelStates {
 
 type IDisplayTextMapping = {
   toasts: Record<EToastType, string>;
+  getToastError: (arg0: (arg0: string) => string) => string;
   routeNames: Record<ERoutesNames, string>;
   buttons: Record<EButtonTexts, string>;
   boolean: Record<EBoolean, string>;
   tableStates: Record<ETableStates, string>;
   filterPanel: Record<EFilterPanelStates, string>;
+  toolBar: Record<EToolbarText, ((arg0: string) => string) | string>;
```

Comment:
i'm not sure why we need this union type here... you can use function and in case you don't need any params just return the text...
beside that you can have multiple params so you can make the type more general using this approaches (either of them):
```ts
type FooFunc = <T extends Record<string, string>>(args: T) => string;
export const foo: FooFunc = (args) => {
  const { a, b, c } = args;
  return `${a}-${b}-${c}`;
};
foo({ a: 'a', b: 'b', c: 'c' });

type BarFunc = (...args: string[]) => string;
export const bar: BarFunc = (a, b, c) => `${a}-${b}-${c}`;
bar('a', 'b', 'c');
```